commandapi-kotlin/commandapi-velocity-kotlin/src/main/kotlin/dev/jorel/commandapi/kotlindsl/CommandAPICommandDSL.kt

```diff
@@ -8,6 +8,7 @@ import dev.jorel.commandapi.arguments.*
 import java.util.function.Predicate
 
 inline fun commandAPICommand(name: String, command: CommandAPICommand.() -> Unit = {}) = CommandAPICommand(name).apply(command).register()
+inline fun commandAPICommand(name: String, namespace: String, command: CommandAPICommand.() -> Unit = {}) = CommandAPICommand(name).apply(command).register(namespace)
```

Comment:
Yeah, Bukkit has something similar but I didn't deal with it because I don't think it's possible with the way we set up commands.
If you feel like there's a way, feel free to contribute to this branch. Obviously, associating commands with plugins should only happen when the `register` method with the plugin object is used - if it's possible at all.